### DISCUSSION TITLE
Programming naming conventions

### DISCUSSION THREAD

**Discussion by Mohammadreza-73** [milwad-dev/laravel-validate#17] [Ideas]
Hi, @milwad-dev

I think it's good idea to add some naming conventions for programming, like:
- camelCase
- PascalCase
- snake_case
- kebab-case
- etc

I'll be happy to let me add these features to project.

**Comment by milwad-dev** [comment]
Can you explain more

**Reply by Mohammadreza-73** [reply]
I mean add rule for check naming conventions.

for example:
```php
$rules = ['camel_case' => [new ValidCamelCase()]];
$data = ['camel_case' => 'helloWorld'];
$passes = $this->app['validator']->make($data, $rules)->passes();

$this->assertTrue($passes);
```

**Reply by milwad-dev** [reply]
Oh, swite!
You can add PR for these, also I add some rule like this 👍
